app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ReviewsController < ApplicationController
+  def new
+    @review = Review.new
+    @restaurant = Restaurant.find(params[:restaurant_id])
+  end
+
+  def create
+    @review = Review.new(review_params)
+      if @review.save
+        flash[:notice] = 'Review Added Successfully'
+        redirect_to @review.restaurant
+      else
+        flash[:alert] = "error!"
+        render :new
+    end
+  end
+
+  # def edit
+  # end
+  #
+  # def update
+  # end
+  #
+  # def delete
+  # end
```

Comment:
As a general rule, I would prefer not to see commented-out code in the `master` branch (better to delete it, if you don't intend to use it.)